lstchain/image/muon/muon_analysis.py

```diff
@@ -394,14 +409,16 @@ def create_muon_table():
             'muon_efficiency': [],
             'ring_containment': [],
             'ring_completeness': [],
-            'ring_pixel_completeness': [],
+            #  missing in ctapipe 0.8:
+            # 'ring_pixel_completeness': [],
```

Comment:
were we using the `ring_pixel_completeness` anywhere anymore? I think this was some remnant from the initial muon code... it may be ok to leave it out, otherwise we should go back to its implementation in `ctapipe`

Reply:
We are not using it as far as I know.